tests/unit/components/cylc/tree/tree.vue.spec.js

```diff
@@ -23,8 +23,27 @@ import Vuetify from 'vuetify/lib'
 import Tree from '@/components/cylc/tree/Tree'
 import { simpleWorkflowTree4Nodes } from './tree.data'
 import TaskState from '@/model/TaskState.model'
+import TreeItem from '@/components/cylc/tree/TreeItem'
+import CylcObjectPlugin from '@/components/cylc/cylcObject/plugin'
 
 const localVue = createLocalVue()
+localVue.prototype.$eventBus = {
+  emit: () => {}
+}
+localVue.prototype.$workflowService = {
+  register: function () {
+  },
+  unregister: function () {
+  },
+  subscribe: function () {
+  },
+  mutations: [
+    {
+      args: []
+    }
+  ]
+}
+localVue.use(CylcObjectPlugin)
```

Comment:
The code above removes several warnings from the console about missing the directive for `cylc-object`.